src/biip/_parser.py

```diff
@@ -90,6 +96,29 @@ def parse(
         raise ParseError(f"Failed to parse {value!r}:\n{result._get_errors_list()}")
 
 
+def parse_hri(value: str) -> ParseResult:
```

Comment:
I don't think I want this to be a top-level function in Biip, as it is specific to GS1 Element Strings. How about moving this into the `GS1Message` class as a class method next to `parse()`?

Reply:
You know best ;)

Will let you know when I'm done the changes.